packages/docs/docs/about.mdx

```diff
@@ -1,39 +1,67 @@
 ---
-id: my-home-doc
+id: about
 sidebar_position: 1
 slug: /
 ---
 
-# About AI.JSX
+# Welcome to AI.JSX!
 
 <p align="center">
-  <img src="img/ai-jsx.svg" alt="AI.JSX logo" width="300" />
+  <img src="img/ai-jsx.svg" alt="AI.JSX logo" width="200" />
 </p>
 
-AI.JSX is a framework that helps you build [AI-powered apps](./guides/brand-new.md). It aims to solve the following problems for developers:
+[![Docs Site](https://img.shields.io/badge/Docs%20Site-docs.ai--jsx.com-orange)](https://docs.ai-jsx.com)
+[![Discord Follow](https://dcbadge.vercel.app/api/server/MsKAeKF8kU?style=flat)](https://discord.gg/MsKAeKF8kU)
+[![Twitter Follow](https://img.shields.io/twitter/follow/fixieai?style=social)](https://twitter.com/fixieai)
 
-- “How do I get data into my LLM call?”
-- “How do I easily switch between model providers?”
-- “How do I performantly chain the results of LLM calls together?”
-- “How do I take advantage of community tools + patterns (e.g. chain of thought)?”
-- “How do I make the model use tools?”
-- “How do I constrain the model’s output?”
-- “How do I structure my app in a post-AI world?”
+AI.JSX is a framework for building AI applications using
+[JSX](https://react.dev/learn/writing-markup-with-jsx). With AI.JSX,
+you don't just use JSX to describe what your UI should look like,
+you also use it to describe how **Large Language Models**, such as
+ChatGPT, should integrate into the rest of your application. The
+end result is a powerful combination where _intelligence_ can be
+deeply embedded into the application stack.
 
-JSX provides a powerful substrate on which to build apps that address these needs.
+AI.JSX is designed to give you two important capabilities out of the box:
+
+1. An intuitive mechanism for orchestrating multiple LLM calls
+   expressed as modular, re-usable components.
+1. The ability to seamlessly interweave UI components with your AI components. This
+   means you can rely on the LLM to construct your UI dynamically from
+   a set of standard React components.
+
+## Getting started
+
+To get started, follow the [Getting Started Guide](./getting-started), or check out the
+[examples](https://github.com/fixie-ai/ai-jsx/tree/main/packages/examples).
+
+AI.JSX can be used to create standalone LLM applications that can
+be deployed anywhere Node.JS is supported, or it can be used as
+part of a larger React application. We have examples that use
+[Next.js](https://github.com/fixie-ai/ai-jsx/tree/main/packages/nextjs-demo)
+and [Create React
```

Comment:
For launch, we're just focusing on Next.js as the supported avenue. CRA will follow shortly thereafter.

I intend to do another pass on the ai-ui doc, and I intend to expand in more detail about CRA being experimental for now.